google/resource_cloudfunctions_function.go

```diff
@@ -530,11 +530,17 @@ func expandEventTrigger(configured []interface{}, project string) *cloudfunction
 		shape = "projects/%s/buckets/%s"
 	case strings.HasPrefix(eventType, "google.pubsub.topic."):
 		shape = "projects/%s/topics/%s"
+	case strings.HasPrefix(eventType, "google.firestore.document."):
+		shape = "projects/%s/databases/(default)/documents/%s"
 	// Legacy style triggers
 	case strings.HasPrefix(eventType, "providers/cloud.storage/eventTypes/"):
 		shape = "projects/%s/buckets/%s"
 	case strings.HasPrefix(eventType, "providers/cloud.pubsub/eventTypes/"):
 		shape = "projects/%s/topics/%s"
+	case strings.HasPrefix(eventType, "providers/cloud.firestore/eventTypes/"):
+		shape = "projects/%s/databases/(default)/documents/%s"
```

Comment:
DO you mind adding a note that we expect `(default)` here? Linking to https://cloud.google.com/functions/docs/calling/cloud-firestore#deploying_your_function

Reply:
I have added a comment.